inc/templates.php

```diff
@@ -191,11 +195,12 @@ function filter_template_loader() {
 /**
  * Return a located template file.
  *
- * @param array $templates A list of possible template files to load.
+ * @param array  $templates A list of possible template files to load.
+ * @param string $folder    Subfolder to look in.
  * @return string The path to the found template.
  */
-function locate_template( array $templates ): string {
-	$template_path = STYLESHEETPATH . '/templates/';
+function locate_template( array $templates, $folder = '/templates/' ): string {
```

Comment:
Rather than passing `$folder` I think we should have separate functions for loading templates and loading template partials so we can handle them separately. A `load_template_part()` function could simply look in the `template-parts` directory for the existence of the specific file being loaded and return the components, rather than resolving the existence of a file from an array of possibilities.

Reply:
Added another function to specifically handle template parts as you suggest. It doesn't feel very dry, but let me know your thoughts.